File: exercice 3.py
"""Quand on exécute truc(10), la fonction affiche 10 et renvoie 20. 
La fonction s’arrête après le premier return et n’exécute pas les instructions suivantes.
Donc, 30 et 40 ne sont pas affichés.
Quand on exécute bidule(10), la fonction affiche 10 et 20, puis renvoie 30.
La fonction s’arrête également après le premier return et n’exécute pas l’instruction suivante.
Donc, 40 n’est pas affiché"""
def truc(x):
    print(x)
    return(2*x)
    print(3*x)
    return(4*x)

def bidule(x):
    print(x)
    print(2*x)
    return(3*x)
    print(4*x) 

File: exercice3.php
<?php 

function truc($x){
    echo ($x);
    return $x*2;
    echo ($x*3);
    return $x*2;
}
truc(10); #la fonction truc renvoie 10 ça ne prend qu' en compte la fonction echo qui vient avant return


function bidule($x){
    echo ($x);
    echo ($x*2);
    return $x*3;
    echo ($x*4);

}
bidule(10); #la fonction bidule renvoie 10 et 20  ça ne prend qu' en compte les fonctions echo qui viennent  avant return
 


?>